packages/stark-ui/src/modules/table/entities/table-row-actions.intf.ts

```diff
@@ -0,0 +1,12 @@
+import { StarkActionBarConfig } from "@nationalbankbelgium/stark-ui";
+
+/**
+ * StarkTableRowActions interface
+ */
+export interface StarkTableRowActions extends StarkActionBarConfig {
+	/**
+	 * Should the actions always be shown on the right side of the table (sticky)
```

Comment:
Maybe we should change "always be shown on the right side"  by "be fixed to the right side of the table"